cmd/block-generator/generator/server.go

```diff
@@ -31,7 +31,20 @@ func initializeConfigFile(configFile string) (config GenerationConfig, err error
 }
 
 // MakeServer configures http handlers. Returns the http server.
-func MakeServer(configFile string, addr string) *http.Server {
+func MakeServer(configFile string, addr string) (*http.Server, Generator) {
+	noOp := func(next http.Handler) http.Handler {
+		return next
+	}
+	return MakeServerWithMiddleware(configFile, addr, noOp)
+}
+
+// BlocksMiddleware is a middleware for the blocks endpoint.
+type BlocksMiddleware func(next http.Handler) http.Handler
```

Comment:
I needed a way to stop the `/v2/blocks` endpoint while leaving the `/v2/accounts` endpoint active. Maybe there is a cleaner way, but it's nice that this middleware approach means the test runner doesn't need to know anything about the server.